.github/actions/firebase-trigger/action.yml

```
@@ -0,0 +1,29 @@
+name: 'Github trigger'
+description: 'Gets the data from other action and changes the firestore'
+inputs:
+  credentials:
+    description: 'Firebase credentials'
+    required: true
+    
+  collection:  # id of input
+    description: 'name of the collection the data is being added'
+    required: true
+    default: 'filters'
```

```suggestion
    default: 'database'
```
we have new structure